pkg/kademlia/kademlia.go

```diff
@@ -301,6 +305,66 @@ func GetIntroNode(addr string) (*pb.Node, error) {
 	}, nil
 }
 
+//refresh updates each Kademlia bucket not contacted in the last hour
+func (k *Kademlia) refresh(ctx context.Context) error {
+	bIDs, err := k.routingTable.GetBucketIds()
+	if err != nil {
+		return Error.Wrap(err)
+	}
+	now := time.Now()
+	startID := bucketID{}
+	for _, bID := range bIDs {
+		ts, tErr := k.routingTable.GetBucketTimestamp(bID)
+		if tErr != nil {
+			err = errs.Combine(tErr)
```

Comment:
This doesn't actually do anything too useful. Maybe use `errs.Group` instead, so:

```
var errors errs.Group

for ... {
    ts, terr := ...
    if terr != nil {
       errors.Add(terr)
       continue
    }
...

return errors.Err()
```